### DISCUSSION TITLE
How to create and set networks in yaml files?

### DISCUSSION THREAD

**Discussion by MartinX3** [containers/podman#12852] [Q&A]
How to create a podman network with `podman play kube`
How to tell a pod to communicate with a specific network declared in the yaml?

I only see examples telling me to create a network in terminal and to use the networkname as a parameter of the `kubernetes play kube` command.

**Comment by rhatdan** [comment]
@umohnani8 PTAL

**Comment by HidingCherry** [comment]
I think this has a clue for that?
https://docs.podman.io/en/latest/markdown/podman-generate-kube.1.html
`kind: Service`

Although, this `kind` is not being listed as supported in:
https://docs.podman.io/en/latest/markdown/podman-play-kube.1.html

kubernetes reference: https://kubernetes.io/docs/reference/kubernetes-api/service-resources/service-v1/
kubernetes concept: https://kubernetes.io/docs/concepts/services-networking/service/

**Comment by MartinX3** [comment]
Is it a feature on the todo list?

**Reply by mheon** [reply]
Can you make a Github issue for this? Discussions are good for questions, but feature requests are a lot easier to track as issues.

**Reply by MartinX3** [reply]
Oh, I didn't know, that it is a missing feature. I just thought I don't find the right documentation. Thank you.

**Reply by MartinX3** [reply]
Reference: https://github.com/containers/podman/issues/12965

**Comment by rhatdan** [comment]
@mheon @baude @Luap99 PTAL

**Comment by fsdrw08** [comment]
Any updates?
